griptape/drivers/web_search/google_web_search_driver.py

```diff
@@ -11,24 +11,24 @@
 
 @define
 class GoogleWebSearchDriver(BaseWebSearchDriver):
-    api_key: str = field(default=None, kw_only=True)
-    search_id: str = field(default=None, kw_only=True)
+    api_key: str = field(kw_only=True)
+    search_id: str = field(kw_only=True)
```

Comment:
Why were they not optional before? Did it just result in a runtime error?

Reply:
Not sure why they were optional, as far as I can tell it just did not work without them.